src/Repository/UserRepository.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types=1);
+
+namespace App\Repository;
+
+use App\Entity\User;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+/**
```

Comment:
Remove this, see discussion with @ayumiesan. She can help you with that

Reply:
ok I see the discussion you had about using @method as annotation